packages/monaco/package.json

```diff
@@ -52,6 +52,7 @@
     "@monaco-editor/loader": "^1.3.3",
     "@monaco-editor/react": "^4.5.1",
     "@repo/ui": "workspace:^",
+    "@vercel/analytics": "^1.0.2",
```

Comment:
a bit sus I have to add this here since it's in a separate package.

Reply:
you can pass the tracker through from the web package where we consume the component. i think the editor takes callbacks we define for both tests and user code. can just put the tracker i believe

Reply:
for now I just only run `track` in the user face panel.